recipes/anaconda-clean/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "anaconda-clean" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ContinuumIO/anaconda-clean/archive/{{ version }}.tar.gz
+  sha256: 2e06c45ed754aa5601afd3ff674bd269e1233c86aedd3ff1c80ef3dc3161d347
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

```suggestion
  number: 0
  entry_points:
    - clean = anaconda_clean:main
```

wow, such a terrible command name....

---

I was looking at that but wasn't sure to include it because it's not defined as an entry point.

And yeah, that can _easily_ be accidentally misused.